src/engine/renderer/tr_init.cpp

```diff
@@ -1063,6 +1064,7 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 		r_depthbits = ri.Cvar_Get( "r_depthbits", "0",  CVAR_LATCH );
 		r_ext_multisample = ri.Cvar_Get( "r_ext_multisample", "0",  CVAR_LATCH | CVAR_ARCHIVE );
 		r_mode = ri.Cvar_Get( "r_mode", "-2", CVAR_LATCH | CVAR_ARCHIVE );
+		r_noBorder = ri.Cvar_Get( "r_noBorder", "0", CVAR_ARCHIVE );
```

Comment:
It seems to me, it should be CVAR_LATCH.

Reply:
I agree but I remember `CVAR_LATCH` can't be set at startup yet (requires a restart after startup). We got the problem wiht `fs_legacyPaks` before.

This option is very likely to be set on command line of a launcher (users already know how to do it for other games like valve's ones).

Reply:
It works fine for all the other window sizing cvars like `r_fullscreen` etc., so why is this one a problem?
Pak loading happens earlier than renderer initialization.

Reply:
Well, `r_fullscreen` is currently not a `CVAR_LATCH`… this may have fooled me to think it would be like that early filesystem cvar…

Experience proves both `r_noBorder` and `r_fullscreen` can be `CVAR_LATCH`.

I flagged properly `r_noBorder` and added a commit to also flag `r_fullscreen`.

Reply:
lol, now that `r_fullscreen` is `CVAR_LATCH`, pressing Alt+Enter does not toggle fullscreen…

Reply:
So, after some investigations I figured that `r_noBorder` can be applied immediately on change (like `r_fullscreen`, so there I removed `CVAR_LATCH` flag and added code to apply it on change (like it is done for `r_fullscreen`).

While I was at it, I edited a bit the code to toggle fullscreen on `r_fullscreen` change, because it was using the cvar value as fullscreen flag, which was working by luck and was not raising any compiler because both were integer, but was wrong.